index.src.html

```diff
@@ -678,7 +687,14 @@
             {{VideoFramePlaneInit/cropHeight}}
             be the crop region of the decoded video frame |output| in
             pixels, prior to any aspect ratio adjustments.
-        5. Let {{VideoFramePlaneInit/displayWidth}} and
+        5. If |displayAspectWidth| and |displayAspectWidth| are provided,
```

Comment:
Change the second `displayAspectWidth` by `Height`.

Reply:
Done.